src/Templates/Microsoft.CodeAnalysis.AnalyzerTemplate/MyProject.Analyzer.CodeFixes/MyProject.Analyzer.CodeFixes.csproj

```diff
@@ -0,0 +1,22 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+    <IsPackable>false</IsPackable>
+    <RootNamespace>MyProject</RootNamespace>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.CodeAnalysis.CSharp.Workspaces" Version="$(MicrosoftCodeAnalysisCSharpVersionVS2019)" />
```

Comment:
Wondering that maybe `Microsoft.CodeAnalysis.CSharp` dependency is enough?

Reply:
Analyzers can reference just the compiler (Microsoft.CodeAnalysis.CSharp), but code fixes require access to the workspaces layer.

Reply:
Got it, thanks